src/popup/components/AccountSelector.vue

```diff
@@ -89,6 +107,20 @@ export default defineComponent({
     margin-bottom: 4px;
     margin-left: -3px; // Compensate roundness
     color: $color-white;
+
+    &.avatar-only {
+      margin-bottom: 0;
+
+      &.btn-pill {
+        padding: 4px;
+        background-color: transparent;
+
+        & .button-plain {
+          width: 24px;
+          height: 24px;
+        }
+      }
+    }
```

Comment:
Parent components should not change the child component's inner styling. If you want the BtnPill to behave differently use bool props/variants.